### PR TITLE
Option for reporting details of pending specs

### PR DESCRIPTION
**Pull request by geekdave**
This is useful when there are a very large number of specs, and you want to easily identify any "technical debt" specs that have not yet been implemented, without having to scroll through a large report and find them individually.

### CODE REVIEW

File: src/unfunk.ts [hunk 1]
```diff
@@ -419,6 +422,17 @@ module unfunk {
 					}
 
 					//details
+                    if(options.reportPending && stats.pending > 0) {
+                        out.writeln(style.accent('->') + ' reporting ' + style.warn(pluralize('pending spec', pending.length)));
+                        out.writeln();
+                        pending.forEach(function (test, num) {
```
Comment (Bartvds) [inline src/unfunk.ts]: Shouldn't this be `if(options.reportPending && stats.pending.length > 0) {` ? (note the `.length`)

Also to keep the style idiomatic this should use a arrow function: `pending.forEach((test, num) => {`

**Comment by Bartvds** [comment]
Thanks for sending this pull request. I'm happy to merge this if a few things are fixed:
1. See the note I added about the if and arrow functions.
2. Indentation looks off: I think I used tabs here: use git's diff viewer to verify consistency.
3. Enable the new option in the `Gruntfile` (about [here](https://github.com/Bartvds/mocha-unfunk-reporter/blob/1826b780855b22484f0cb9dfd9f60e8e28494af7/Gruntfile.js#L159)). 
4. Then add a simple `pending.test.ts` demo test file and add it to the `typescript:test_pass` target. Then it will be run by `grunt test` so I can see it in action in the travis build report. 

(I'm aware the project needs more solid testing; I have a future update planned that will verify the actual console output)

**Comment by geekdave** [comment]
Thanks for the feedback.  I think I've resolved all the issues.  This was my first time working with TypeScript so I might have gotten some things wrong. :)

**Comment by Bartvds** [comment]
Thanks, looks good. I've merged and will try it out in my own projects. I'll push it to NPM in next update together with my own fixes.

This is also one of my own try outs with TypeScript to be honest. I like it so far, its a lot like ActionScript 3 and planned ES6.

**Comment by Bartvds** [comment]
I pushed this live in v0.2.1

**Pull request merged**